### ISSUE TITLE
Unify mobile rum disk caching configuration

### ISSUE THREAD

**Issue by mhennoch** [signalfx/gdi-specification#264]
Currently:
 
Android
enableDiskBuffering
limitDiskUsageMegabytes

iOS
enableDiskCache
spanDiskCacheMaxSize

Android limit is set in MB and iOS one in bytes.
Default limit for both seems to be 25mb so that is fine.
Lets unify the configuration options? React Native and flutter will need those also.

**Comment by pellared** [comment]
Should it not be defined under https://github.com/signalfx/gdi-specification/blob/main/specification/configuration.md#real-user-monitoring-libraries?

**Comment by m-tu** [comment]
I would agree. I will have do come up with config options for RN soon will propose something then.
